tests/filecheck/mlir-conversion/with-mlir/mlir_opt.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: xdsl-opt %s -p mlir-opt{arguments='--cse','--mlir-print-op-generic'} --print-op-generic | filecheck %s
```

Comment:
Not sure how mlir structures it's optimizations, but this feels more like dce than cse. Maybe change the example (or ignore the comment 😉 )

Reply:
```
mlir-opt: Unknown command line argument '--dce'.  Try: 'mlir-opt --help'
mlir-opt: Did you mean '--cse'?
```